src/MSALWrapper/AuthFlow/AuthFlowExecutor.cs

```diff
@@ -73,5 +85,54 @@ public async Task<IEnumerable<AuthFlowResult>> GetTokenAsync()
 
             return resultList;
         }
+
+        /// <summary>
+        /// Run the auth mode in a separate task and
+        /// poll to see if we hit global timeout before the auth flow completes.
+        /// </summary>
+        /// <param name="authFlow">Auth Flow that we are executing.</param>
+        /// <returns>AuthFlowResult containing Error if CLI times out; else actual result of the AuthFlow.</returns>
+        private async Task<AuthFlowResult> GetTokenAndPoll(IAuthFlow authFlow)
+        {
+            var flowResult = Task.Run(() => authFlow.GetTokenAsync());
+            var authFlowName = authFlow.GetType().Name;
+            var oneMinute = TimeSpan.FromMinutes(1);

```

Comment:
Why one minutes. It seems like another magic number.

Reply:
Yes, I can move that with the other constant and that should provide a better context but in essence that is for waiting before we start giving warning to user